MdeModulePkg/Library/DxeCapsuleLibFmp/DxeCapsuleProcessLibNull.c

```diff
@@ -68,3 +68,25 @@ ProcessCapsules (
 {
   return EFI_UNSUPPORTED;
 }
+
+// MS_CHANGE [BEGIN] - This implementation is BootServices/Runtime specific.

```

Comment:
Based on existing comments, I know you're going to look at this now, but this is another instance.

```suggestion
// MU_CHANGE [BEGIN] - This implementation is BootServices/Runtime specific.
```